tests/capture/test_base_interpreter.py

```diff
@@ -543,3 +543,50 @@ def circuit(y):
         assert qfunc_jaxpr.eqns[1].primitive == qml.RX._primitive  # eqn 0 is mul
         assert qfunc_jaxpr.eqns[2].primitive == qml.Z._primitive
         assert qfunc_jaxpr.eqns[3].primitive == qml.ops.SProd._primitive
+
+
+@pytest.mark.usefixtures("enable_disable_dynamic_shapes")
+class TestDynamicShapes:
+    """Test that our interpreters can handle dynamic array creation."""
+
+    @pytest.mark.parametrize("reinterpret", (True, False))
+    def test_creating_ones(self, reinterpret):
+        """Test that broadcast_in_dim can be executed with PlxprInterpreter."""
```

Comment:
Maybe this test can be parametrized over `jnp` functions that we know to use the `broadcast_in_dim` primitive (`jnp.ones`, `jnp.zeros`, `jnp.full`)? Just to cover our bases.

Reply:
added